### PR TITLE
DOC: setup.py: url, project_urls (Source, Docs, Issues)

### PR DESCRIPTION
**Pull request by westurner**
See: https://github.com/certifi/certifi.io/issues/16 
- https://certifi.io/ is currently down

### CODE REVIEW

**Comment by westurner** [comment]
Warehouse now supports ``project_urls=`` in setup.py metadata (PEP 345 Project-URL metadata https://github.com/pypa/warehouse/issues/1419)

**Comment by hugovk** [comment]
Any ideas when https://certifi.io will be back up?

It's been down since at least 9 months, since [November 2017](https://github.com/certifi/certifi.io/issues/16).

I'd suggest changing it GitHub and then changing it back if/when https://certifi.io goes back up.

Thank you!

**Comment by westurner** [comment]
ReadTheDocs now supports HTTPS and HSTS on custom *CNAME* (sub)domains.

https://github.com/rtfd/readthedocs.org/issues/2652

Cryptography.io is also hosting on a bare (so non-CNAME) domain.

**Comment by jdufresne** [comment]
https://certifi.io/en/latest/ is back up.

**Comment by westurner** [comment]
Thanks!

Looks like the CA bundle download link now points to
https://mkcert.org/generate/

**Comment by benoitc** [comment]
> https://certifi.io/ is back up.

hmmm that happeneda long time ago. or did I miss something?

**Comment by jdufresne** [comment]
Oh, perhaps it did. I only just noticed while browsing this issue and thought I'd add an update to remove any lingering confusion.

File: setup.py [hunk 1]
```diff
@@ -64,4 +64,9 @@
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
     ),
+    project_urls={
+        'Source Code': 'https://github.com/certifi/python-certifi',
```
Comment (hugovk) [inline setup.py]: "Source" is about three times as common as "Source Code": https://github.com/hugovk/pypi-tools/issues/11#issuecomment-549175199

```suggestion
        'Source': 'https://github.com/certifi/python-certifi',
```

File: setup.py [hunk 2]
```diff
@@ -64,4 +64,9 @@
...
+        'Documentation': 'https://certifiio.readthedocs.io/en/latest/',
+        'Bug Tracker': 'https://github.com/certifi/python-certifi/issues',
```
Comment (hugovk) [inline setup.py]: "Issue [tT]racker" is about four times as common as "Bug Tracker": https://github.com/hugovk/pypi-tools/issues/11#issuecomment-549175199

```suggestion
        'Issue Tracker': 'https://github.com/certifi/python-certifi/issues',
```

File: setup.py [hunk 3]
```diff
@@ -39,7 +39,7 @@
     long_description=open('README.rst').read(),
     author='Kenneth Reitz',
     author_email='me@kennethreitz.com',
-    url='http://certifi.io/',
+    url='https://github.com/certifi/python-certifi',
```
Comment (hugovk) [inline setup.py]: Now it's back up, use the original URL with HTTPS:
```suggestion
    url='https://certifi.io/',
```

**Pull request closed without merging**